src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/mds-summary.pipe.ts

```diff
@@ -42,22 +42,49 @@ export class MdsSummaryPipe implements PipeTransform {
       contentLine1 = `${active} ${this.i18n('active')}`;
       contentLine2 = `${standbys + standbyReplay} ${this.i18n('standby')}`;
     }
-
+    let standbyHoverText = '';
+    let standbyflg = false;
+    for (const i in value.standbys) {
+      if (standbyflg) {
+        standbyHoverText = standbyHoverText + ', ' +  value.standbys[i].name;
+      } else {
+        standbyHoverText = standbyHoverText + value.standbys[i].name;
+        standbyflg = true;
+      }
+    }
```

Comment:
Hi how about the following:
```suggestion
    const standbyHoverText = value.standbys.map((s: any): string => s.name).join(', ');
```

Reply:
standbys doesn't have 'map' property, will show undefined

Reply:
As `value.standbys` holds an array it holds the [build in method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) named `map`.

Reply:
Yes . But the problem is if I make this change (for mds only, working for mgr), the health component spec file is not passing the tests, throwing the error: 
 Cannot read property 'map' of undefined

Reply:
It has been resolved . The method suggested works fine